Formula/phpunit@7.rb

```diff
@@ -0,0 +1,112 @@
+class PhpunitAT7 < Formula
+  desc "Programmer-oriented testing framework for PHP"
+  homepage "https://phpunit.de"
+  url "https://phar.phpunit.de/phpunit-7.5.2.phar"
```

Comment:
Also, you need to update this to v7.5.18

8a5f3954faa792400d82a81763da071b1f61a4772acfcd3b7310b85dd8a5503b